frontend/src/components/CompareAssignmentsTab/CompareAssignmentsTab.js

```diff
@@ -39,6 +43,11 @@ const CompareAssignmentsTab = () => {
       ))
     ) : <p>{state.placeholder}</p>;
 
+  const csvLink = loading
+    ? (
+      <CsvDownloadLink data={state.payload.denormalized_data} text=".CSV Download" />
```

Comment:
Yeah, it depends on how the data is passed to `CSVLink` but it can auto-detect the headers. Since in this instance we are passing it an array of objects, it can detect the headers.